addon-test-support/audit.ts

```diff
@@ -71,67 +124,28 @@ function isNotContextObject(obj: MaybeContextObject) {
  * @method runA11yAudit
  * @private
  */
-function runA11yAudit(
-  contextSelector:
-    | ElementContext
-    | RunOptions
-    | undefined = '#ember-testing-container',
+export default function a11yAudit(
+  contextSelector: MaybeElementContext = '#ember-testing-container',
   axeOptions?: RunOptions | undefined
 ) {
   mark('a11y_audit_start');
 
-  let context: ElementContext;
-  let options: RunOptions | undefined = axeOptions;
-
-  // Support passing axeOptions as a single argument
-  if (arguments.length === 1) {
-    if (isPlainObj(contextSelector) && isNotContextObject(contextSelector)) {
-      context = '#ember-testing-container';
-      options = <RunOptions>contextSelector;
-    } else {
-      context = <ElementContext>contextSelector;
-    }
-  } else {
-    context = <ElementContext>contextSelector;
-  }
-
-  if (!options) {
-    // Try load default config
-    let a11yConfig = config['ember-a11y-testing'] || {};
-    let componentOptions = a11yConfig['componentOptions'] || {};
-    options = componentOptions['axeOptions'] || {};
-  }
+  let [context, options] = _normalizeRunParams(contextSelector, axeOptions);
 
   document.body.classList.add('axe-running');
 
-  let auditPromise = new RSVP.Promise((resolve, reject) => {
-    run(context, options!, (error, result) => {
+  return new RSVP.Promise((resolve, reject) => {
+    run(context, options, (error, result) => {
       if (!error) {
         return resolve(result);
       } else {
         return reject(error);
       }
     });
-  });
-
-  return auditPromise.then(a11yAuditCallback).finally(() => {
-    document.body.classList.remove('axe-running');
-    markEndAndMeasure('a11y_audit', 'a11y_audit_start', 'a11y_audit_end');
-  });
-}
-
-/**
- * A wrapper method to run the async a11yAudit test helper if in an acceptance
- * testing situation, but also supports being used in integration/unit test
- * scenarios.
- *
- * @method a11yAudit
- * @public
- */
-export default function a11yAudit(...args: any[]) {
-  if ((<any>window).a11yAudit) {
-    return (<any>window).a11yAudit(...args);
-  }
```

Comment:
Should probably call this change out in the PR description, this is a "breaking" change (drops support for super old versions of the test harness that were relying on `Ember.Test.registerHelper`). We should totally clean it up (like you did here), just need to call it out explicitly.

Reply:
Done.